cves/CVE-2013-4249.yml

```diff
@@ -194,8 +208,9 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: false
+  note: As this error was due to improper string sanitization, the CVE was not
```

Comment:
Does this vulnerability affect IPC downstream?

Reply:
No it doesn't